mindsdb/__main__.py

```diff
@@ -19,14 +20,26 @@
 from mindsdb.utilities.ps import is_pid_listen_port, get_child_pids
 from mindsdb.utilities.functions import args_parse, get_versions_where_predictors_become_obsolete
 from mindsdb.utilities.with_kwargs_wrapper import WithKWArgsWrapper
-from mindsdb.utilities.log import log
+from mindsdb.utilities import log
 from mindsdb.interfaces.stream.stream import StreamController
 from mindsdb.interfaces.stream.utilities import STOP_THREADS_EVENT
 from mindsdb.interfaces.model.model_controller import ModelController
 from mindsdb.interfaces.database.integrations import IntegrationController
 import mindsdb.interfaces.storage.db as db
 from mindsdb.integrations.utilities.install import install_dependencies
 
+from mindsdb.utilities.fs import create_dirs_recursive
+from mindsdb.utilities.functions import args_parse
+from mindsdb.utilities.telemetry import telemetry_file_exists, disable_telemetry
+
+# is_ray_worker = False
```

Comment:
When there is probability that some block of code is removed by mistake and something wrong without it, I prefer to keep it commented. And will delete it in time of next task for this file.
It is convinient to see what code was there before in case of debuging